draft-ietf-rtcweb-security-arch.xml

```diff
@@ -556,6 +556,163 @@
       </section>
     </section>
 
+    <section title="SDP Identity Attribute" anchor="sec.sdp-id-attr">
+      <t>
+	    The SDP 'identity' attribute is a session-level attribute that is used
+	    by an endpoint to convey its identity assertion to its peer. The identity
+        assertion value is encoded as Base-64, as described in Section 4 of <xref target="RFC4648"/>.
+      </t>
+      <t>
+	    The procedures in this section are based on the assumption that the identity assertion 
+        of an endpoint is bound to the fingerprints of the endpoint. This does not preclude the 
+        definition of alternative means of binding an assertion to the identity, but such means are 
+        outside the scope of this specification.
+      </t>
+      <t>
+	    The semantics of multiple 'identity' attributes within an offer or answer are undefined.
+	    Implementations SHOULD only include a single identity attribute in an
+	    offer or answer and relying parties MAY elect to ignore all but the first
+	    identity attribute.
+      </t>
+      <t>
+        <list style="hanging">
+        <t>
+          Name: identity
+        </t>
+        <t>
+          Value: identity-assertion
+        </t>
+        <t>
+          Usage Level: session
+        </t>
+        <t>
+          Charset Dependent: no
+        </t>
+        <t>
+          Default Value: N/A
+        </t>
+        </list>
+      </t>
+      <figure>
+      <artwork type="inline"><![CDATA[
+Syntax:
+
+  identity-assertion        = identity-assertion-value 
+                              *(SP identity-extension)
+  identity-assertion-value  = base64
+  identity-extension        = extension-name [ "=" extension-value ]
+  extension-name            = token
+  extension-value           = 1*(%x01-09 / %x0b-0c / %x0e-3a / %x3c-ff)
+                              ; byte-string from [RFC4566]
+
+  <ALPHA and DIGIT as defined in [RFC4566]>
+  <base64 as defined in [RFC4566]>
+  
+ Example:
+
+  a=identity:\
+    eyJpZHAiOnsiZG9tYWluIjoiZXhhbXBsZS5vcmciLCJwcm90b2NvbCI6ImJvZ3Vz\
+    In0sImFzc2VydGlvbiI6IntcImlkZW50aXR5XCI6XCJib2JAZXhhbXBsZS5vcmdc\
+    IixcImNvbnRlbnRzXCI6XCJhYmNkZWZnaGlqa2xtbm9wcXJzdHV2d3l6XCIsXCJz\
+    aWduYXR1cmVcIjpcIjAxMDIwMzA0MDUwNlwifSJ9
+    
+  Note that long lines in the example are folded to meet the column
+  width constraints of this document; the backslash ("\") at the end of
+  a line and the carriage return that follows shall be ignored.
+  
+      ]]></artwork>
+       </figure>
+       <t>
+         This specification does not define any extensions for the attribute
+       </t>
+       <t>
+         The identity-assertion value is a JSON <xref target="RFC8259"/> encoded string. The JSON object 
+         contains two keys: "assertion" and "idp". The <spanx style="verb">"assertion"</spanx> key value contains 
```

Comment:
The first mentions of assertion and idp need to be quoted too.

Reply:
Will fix.

Reply:
I did not find the ide and assertion keys mentioned elsewhere.